src/main/java/org/mondemand/Client.java

```diff
@@ -344,12 +344,9 @@ public void addContext(String key, String value) throws MondemandException {
     if(key == null) {
       throw new MondemandException("key is null");
     }
-    if(value == null || value.isEmpty()) {
+    if(value == null) {
       throw new MondemandException("value is empty or null");
```

Comment:
we are ok with empty sting keys?  if so, the exception message should be updated.

Reply:
duh sorry it's a value, not a key... but exception message should still be updated.

Reply:
I was going to ask whether or not we should be ok with `null` key/values too, or not?